main.py

```
@@ -80,21 +77,20 @@ def sync_nonce(state, pending_nonce_map, address):
 async def node_loop():
     logger.info("Starting MiniChain Node with Smart Contracts")
 
-    state = State()
-    chain = Blockchain(state)
+    chain = Blockchain()
     mempool = Mempool()
 
     pending_nonce_map = {}
 
     def claim_nonce(address):
```

Ruff (0.15.1)

[warning] Missing return type annotation for private function `claim_nonce`

(ANN202)